### ISSUE TITLE
Add Focus Indicator to Header

### ISSUE THREAD

**Issue by hannahackermans** [elmcip/elmcip#493]
### WCAG Guidelines 
**Success Criterion 2.4.7 Focus Visible** 

> (Level AA)
> Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible. 

### Current State of ELMCIP
There is a blue rectangle indication of focus on most of the website, but not on the header for logged in users. The contrast to indicate focus is also very low (see #492 )

Cropped screenshot below show that the focus is on "Content" because it makes the button grey (#444444) as opposed to black, which with a 2.16 score is considered poor contrast for all sizes. There is no blue rectangle indicating focus like on the other content on the website.  
![Screenshot 2023-05-24 10 37 15](https://github.com/elmcip/elmcip/assets/18713442/e2366aab-b733-4b90-8a6f-45fc04f9d6b1)

### Proposed changes to meet the success criteria 
Add the blue focus indicator to header in the same way we have on the rest of the website. 

Cropped screenshot below shows the focus is on "Dashboard" because it has a blue rectangle to indicate focus. 
![image](https://github.com/elmcip/elmcip/assets/18713442/ef016a43-bfef-45bc-8e44-71c51d03dedc)
